package.json

```diff
@@ -42,7 +42,7 @@
     "stylelint": "^8.2.0"
   },
   "hexo": {
-    "version": "3.4.0"
+    "version": "3.3.8"
```

Comment:
Is this supposed to be 3.3.8 or 3.4.0? It looks like it was changed back to 3.3.8 in the master branch?

Reply:
Running `npm run build` locally changes it to `3.4.0`. I will amend your commit to include the change.

Reply:
Thank you ^_^

Reply:
I'll also [fix the linting problem](https://travis-ci.org/sonarwhal/sonarwhal.com/builds/306069030#L2275).

Reply:
Thank you, again.